### PR TITLE
Update `popular_actions.go`

### PR DESCRIPTION
**Pull request by mi-wada**
## What

Follow <https://github.com/rhysd/actionlint/blob/main/CONTRIBUTING.md#maintain-popular_actionsgo>

1. `go run ./scripts/generate-popular-actions -d` and fix `scripts/generate-popular-actions/popular_actions.json`
2. `go generate`

## Why

Looks fails auto generate Actions due to major version release.
https://github.com/rhysd/actionlint/actions/runs/12208933028

Due to not updated `popular_actions.go`, actionlint reports false positive on my repo.
image

### CODE REVIEW

File: scripts/generate-popular-actions/popular_actions.json [hunk 1]
```diff
@@ -471,8 +471,8 @@
     },
     {
         "slug": "actions/attest-build-provenance",
-        "tags": ["v1"],
-        "next": "v2"
+        "tags": ["v1", "v2"],
```
Comment (mi-wada) [inline scripts/generate-popular-actions/popular_actions.json:474]: About v2: https://github.com/actions/attest-build-provenance/releases/tag/v2.0.0

File: scripts/generate-popular-actions/popular_actions.json
Comment (mi-wada) [inline scripts/generate-popular-actions/popular_actions.json:1]: After this update, `go run ./scripts/generate-popular-actions -d` finish with 0.

<details>
  <summary>`go run ./scripts/generate-popular-actions -d` result</summary>

```shell
$ go run ./scripts/generate-popular-actions -d
2024/12/09 16:30:53 Start generate-popular-actions script
2024/12/09 16:30:53 Start detecting new versions in 90 repositories
2024/12/09 16:30:53 Checking https://raw.githubusercontent.com/8398a7/action-slack/v4/action.yml
2024/12/09 16:30:53 Checking https://raw.githubusercontent.com/Azure/functions-action/v2/action.yml
2024/12/09 16:30:53 Checking https://raw.githubusercontent.com/Azure/container-scan/v1/action.yaml
2024/12/09 16:30:53 Checking https://raw.githubusercontent.com/EnricoMi/publish-unit-test-result-action/v3/action.yml
2024/12/09 16:30:54 Not found: https://raw.githubusercontent.com/8398a7/action-slack/v4/action.yml
2024/12/09 16:30:54 Checking https://raw.githubusercontent.com/JamesIves/github-pages-deploy-action/v5/action.yml
2024/12/09 16:30:54 Not found: https://raw.githubusercontent.com/EnricoMi/publish-unit-test-result-action/v3/action.yml
2024/12/09 16:30:54 Checking https://raw.githubusercontent.com/ReactiveCircus/android-emulator-runner/v3/action.yml
2024/12/09 16:30:54 Not found: https://raw.githubusercontent.com/Azure/functions-action/v2/action.yml
2024/12/09 16:30:54 Checking https://raw.githubusercontent.com/Swatinem/rust-cache/v3/action.yml
2024/12/09 16:30:54 Not found: https://raw.githubusercontent.com/Azure/container-scan/v1/action.yaml
2024/12/09 16:30:54 Checking https://raw.githubusercontent.com/actions-cool/issues-helper/v4/action.yml
2024/12/09 16:30:54 Not found: https://raw.githubusercontent.com/JamesIves/github-pages-deploy-action/v5/action.yml
2024/12/09 16:30:54 Checking https://raw.githubusercontent.com/actions/cache/v5/action.yml
2024/12/09 16:30:54 Not found: https://raw.githubusercontent.com/ReactiveCircus/android-emulator-runner/v3/action.yml
2024/12/09 16:30:54 Checking https://raw.githubusercontent.com/actions/cache/v5/save/action.yml
2024/12/09 16:30:54 Not found: https://raw.githubusercontent.com/actions-cool/issues-helper/v4/action.yml
2024/12/09 16:30:54 Not found: https://raw.githubusercontent.com/Swatinem/rust-cache/v3/action.yml
2024/12/09 16:30:54 Checking https://raw.githubusercontent.com/actions/checkout/v5/action.yml
2024/12/09 16:30:54 Checking https://raw.githubusercontent.com/actions/cache/v5/restore/action.yml
2024/12/09 16:30:54 Not found: https://raw.githubusercontent.com/actions/cache/v5/action.yml
2024/12/09 16:30:54 Checking https://raw.githubusercontent.com/actions/configure-pages/v6/action.yml
2024/12/09 16:30:54 Not found: https://raw.githubusercontent.com/actions/checkout/v5/action.yml
2024/12/09 16:30:54 Checking https://raw.githubusercontent.com/actions/deploy-pages/v5/action.yml
2024/12/09 16:30:54 Not found: https://raw.githubusercontent.com/actions/cache/v5/save/action.yml
2024/12/09 16:30:54 Checking https://raw.githubusercontent.com/actions/delete-package-versions/v6/action.yml
2024/12/09 16:30:54 Not found: https://raw.githubusercontent.com/actions/cache/v5/restore/action.yml
2024/12/09 16:30:54 Checking https://raw.githubusercontent.com/actions/download-artifact/v5/action.yml
2024/12/09 16:30:55 Not found: https://raw.githubusercontent.com/actions/configure-pages/v6/action.yml
2024/12/09 16:30:55 Checking https://raw.githubusercontent.com/actions/first-interaction/v2/action.yml
2024/12/09 16:30:55 Not found: https://raw.githubusercontent.com/actions/deploy-pages/v5/action.yml
2024/12/09 16:30:55 Not found: https://raw.githubusercontent.com/actions/download-artifact/v5/action.yml
2024/12/09 16:30:55 Checking https://raw.githubusercontent.com/actions/labeler/v6/action.yml
2024/12/09 16:30:55 Checking https://raw.githubusercontent.com/actions/github-script/v8/action.yml
2024/12/09 16:30:55 Not found: https://raw.githubusercontent.com/actions/delete-package-versions/v6/action.yml
2024/12/09 16:30:55 Checking https://raw.githubusercontent.com/actions/setup-dotnet/v5/action.yml
2024/12/09 16:30:55 Not found: https://raw.githubusercontent.com/actions/first-interaction/v2/action.yml
2024/12/09 16:30:55 Checking https://raw.githubusercontent.com/actions/setup-go/v6/action.yml
2024/12/09 16:30:55 Not found: https://raw.githubusercontent.com/actions/setup-dotnet/v5/action.yml
2024/12/09 16:30:55 Not found: https://raw.githubusercontent.com/actions/labeler/v6/action.yml
2024/12/09 16:30:55 Checking https://raw.githubusercontent.com/actions/setup-java/v5/action.yml
2024/12/09 16:30:55 Checking https://raw.githubusercontent.com/actions/setup-node/v5/action.yml
2024/12/09 16:30:55 Not found: https://raw.githubusercontent.com/actions/github-script/v8/action.yml
2024/12/09 16:30:55 Checking https://raw.githubusercontent.com/actions/setup-python/v6/action.yml
2024/12/09 16:30:55 Not found: https://raw.githubusercontent.com/actions/setup-go/v6/action.yml
2024/12/09 16:30:55 Checking https://raw.githubusercontent.com/actions/stale/v10/action.yml
2024/12/09 16:30:55 Not found: https://raw.githubusercontent.com/actions/setup-node/v5/action.yml
2024/12/09 16:30:55 Checking https://raw.githubusercontent.com/actions/upload-artifact/v5/action.yml
2024/12/09 16:30:55 Not found: https://raw.githubusercontent.com/actions/setup-java/v5/action.yml
2024/12/09 16:30:55 Checking https://raw.githubusercontent.com/actions/upload-pages-artifact/v4/action.yml
2024/12/09 16:30:55 Not found: https://raw.githubusercontent.com/actions/setup-python/v6/action.yml
2024/12/09 16:30:55 Checking https://raw.githubusercontent.com/actions/dependency-review-action/v5/action.yml
2024/12/09 16:30:56 Not found: https://raw.githubusercontent.com/actions/upload-pages-artifact/v4/action.yml
2024/12/09 16:30:56 Checking https://raw.githubusercontent.com/aws-actions/configure-aws-credentials/v5/action.yml
2024/12/09 16:30:56 Not found: https://raw.githubusercontent.com/actions/dependency-review-action/v5/action.yml
2024/12/09 16:30:56 Checking https://raw.githubusercontent.com/azure/aks-set-context/v5/action.yml
2024/12/09 16:30:56 Not found: https://raw.githubusercontent.com/actions/upload-artifact/v5/action.yml
2024/12/09 16:30:56 Checking https://raw.githubusercontent.com/azure/login/v3/action.yml
2024/12/09 16:30:56 Not found: https://raw.githubusercontent.com/actions/stale/v10/action.yml
2024/12/09 16:30:56 Checking https://raw.githubusercontent.com/bahmutov/npm-install/v2/action.yml
2024/12/09 16:30:56 Not found: https://raw.githubusercontent.com/aws-actions/configure-aws-credentials/v5/action.yml
2024/12/09 16:30:56 Checking https://raw.githubusercontent.com/codecov/codecov-action/v6/action.yml
2024/12/09 16:30:56 Not found: https://raw.githubusercontent.com/azure/aks-set-context/v5/action.yml
2024/12/09 16:30:56 Checking https://raw.githubusercontent.com/dawidd6/action-download-artifact/v8/action.yml
2024/12/09 16:30:56 Not found: https://raw.githubusercontent.com/azure/login/v3/action.yml
2024/12/09 16:30:56 Checking https://raw.githubusercontent.com/dawidd6/action-send-mail/v5/action.yml
2024/12/09 16:30:56 Not found: https://raw.githubusercontent.com/bahmutov/npm-install/v2/action.yml
2024/12/09 16:30:56 Checking https://raw.githubusercontent.com/dessant/lock-threads/v6/action.yml
2024/12/09 16:30:56 Not found: https://raw.githubusercontent.com/dawidd6/action-download-artifact/v8/action.yml
2024/12/09 16:30:56 Checking https://raw.githubusercontent.com/docker/build-push-action/v7/action.yml
2024/12/09 16:30:56 Not found: https://raw.githubusercontent.com/dawidd6/action-send-mail/v5/action.yml
2024/12/09 16:30:56 Checking https://raw.githubusercontent.com/docker/login-action/v4/action.yml
2024/12/09 16:30:56 Not found: https://raw.githubusercontent.com/codecov/codecov-action/v6/action.yml
2024/12/09 16:30:56 Checking https://raw.githubusercontent.com/docker/metadata-action/v6/action.yml
2024/12/09 16:30:56 Not found: https://raw.githubusercontent.com/dessant/lock-threads/v6/action.yml
2024/12/09 16:30:56 Checking https://raw.githubusercontent.com/docker/setup-buildx-action/v4/action.yml
2024/12/09 16:30:57 Not found: https://raw.githubusercontent.com/docker/build-push-action/v7/action.yml
2024/12/09 16:30:57 Checking https://raw.githubusercontent.com/docker/setup-qemu-action/v4/action.yml
2024/12/09 16:30:57 Not found: https://raw.githubusercontent.com/docker/setup-buildx-action/v4/action.yml
2024/12/09 16:30:57 Checking https://raw.githubusercontent.com/dorny/paths-filter/v4/action.yml
2024/12/09 16:30:57 Not found: https://raw.githubusercontent.com/docker/metadata-action/v6/action.yml
2024/12/09 16:30:57 Checking https://raw.githubusercontent.com/enriikke/gatsby-gh-pages-action/v3/action.yml
2024/12/09 16:30:57 Not found: https://raw.githubusercontent.com/docker/login-action/v4/action.yml
2024/12/09 16:30:57 Checking https://raw.githubusercontent.com/erlef/setup-beam/v2/action.yml
2024/12/09 16:30:57 Not found: https://raw.githubusercontent.com/docker/setup-qemu-action/v4/action.yml
2024/12/09 16:30:57 Checking https://raw.githubusercontent.com/game-ci/unity-builder/v5/action.yml
2024/12/09 16:30:57 Not found: https://raw.githubusercontent.com/dorny/paths-filter/v4/action.yml
2024/12/09 16:30:57 Checking https://raw.githubusercontent.com/github/codeql-action/v4/analyze/action.yml
2024/12/09 16:30:57 Not found: https://raw.githubusercontent.com/enriikke/gatsby-gh-pages-action/v3/action.yml
2024/12/09 16:30:57 Checking https://raw.githubusercontent.com/github/codeql-action/v4/autobuild/action.yml
2024/12/09 16:30:57 Not found: https://raw.githubusercontent.com/erlef/setup-beam/v2/action.yml
2024/12/09 16:30:57 Checking https://raw.githubusercontent.com/github/codeql-action/v4/init/action.yml
2024/12/09 16:30:57 Not found: https://raw.githubusercontent.com/game-ci/unity-builder/v5/action.yml
2024/12/09 16:30:57 Checking https://raw.githubusercontent.com/github/super-linter/v8/action.yml
2024/12/09 16:30:57 Not found: https://raw.githubusercontent.com/github/codeql-action/v4/analyze/action.yml
2024/12/09 16:30:57 Checking https://raw.githubusercontent.com/githubocto/flat/v4/action.yml
2024/12/09 16:30:57 Not found: https://raw.githubusercontent.com/github/codeql-action/v4/autobuild/action.yml
2024/12/09 16:30:57 Checking https://raw.githubusercontent.com/golangci/golangci-lint-action/v7/action.yml
2024/12/09 16:30:57 Not found: https://raw.githubusercontent.com/github/codeql-action/v4/init/action.yml
2024/12/09 16:30:57 Checking https://raw.githubusercontent.com/google-github-actions/auth/v3/action.yml
2024/12/09 16:30:57 Not found: https://raw.githubusercontent.com/github/super-linter/v8/action.yml
2024/12/09 16:30:57 Checking https://raw.githubusercontent.com/google-github-actions/get-secretmanager-secrets/v3/action.yml
2024/12/09 16:30:57 Not found: https://raw.githubusercontent.com/githubocto/flat/v4/action.yml
2024/12/09 16:30:57 Checking https://raw.githubusercontent.com/google-github-actions/setup-gcloud/v3/action.yml
2024/12/09 16:30:58 Not found: https://raw.githubusercontent.com/golangci/golangci-lint-action/v7/action.yml
2024/12/09 16:30:58 Checking https://raw.githubusercontent.com/google-github-actions/upload-cloud-storage/v3/action.yml
2024/12/09 16:30:58 Not found: https://raw.githubusercontent.com/google-github-actions/auth/v3/action.yml
2024/12/09 16:30:58 Checking https://raw.githubusercontent.com/goreleaser/goreleaser-action/v7/action.yml
2024/12/09 16:30:58 Not found: https://raw.githubusercontent.com/google-github-actions/get-secretmanager-secrets/v3/action.yml
2024/12/09 16:30:58 Checking https://raw.githubusercontent.com/gradle/wrapper-validation-action/v4/action.yml
2024/12/09 16:30:58 Not found: https://raw.githubusercontent.com/google-github-actions/setup-gcloud/v3/action.yml
2024/12/09 16:30:58 Checking https://raw.githubusercontent.com/haskell/actions/v3/setup/action.yml
2024/12/09 16:30:58 Not found: https://raw.githubusercontent.com/google-github-actions/upload-cloud-storage/v3/action.yml
2024/12/09 16:30:58 Checking https://raw.githubusercontent.com/microsoft/playwright-github-action/v2/action.yml
2024/12/09 16:30:58 Not found: https://raw.githubusercontent.com/goreleaser/goreleaser-action/v7/action.yml
2024/12/09 16:30:58 Checking https://raw.githubusercontent.com/mikepenz/release-changelog-builder-action/v6/action.yml
2024/12/09 16:30:58 Not found: https://raw.githubusercontent.com/gradle/wrapper-validation-action/v4/action.yml
2024/12/09 16:30:58 Checking https://raw.githubusercontent.com/msys2/setup-msys2/v3/action.yml
2024/12/09 16:30:58 Not found: https://raw.githubusercontent.com/haskell/actions/v3/setup/action.yml
2024/12/09 16:30:58 Checking https://raw.githubusercontent.com/ncipollo/release-action/v2/action.yml
2024/12/09 16:30:58 Not found: https://raw.githubusercontent.com/microsoft/playwright-github-action/v2/action.yml
2024/12/09 16:30:58 Checking https://raw.githubusercontent.com/nwtgck/actions-netlify/v4/action.yml
2024/12/09 16:30:58 Not found: https://raw.githubusercontent.com/mikepenz/release-changelog-builder-action/v6/action.yml
2024/12/09 16:30:58 Checking https://raw.githubusercontent.com/octokit/request-action/v3.x/action.yml
2024/12/09 16:30:58 Not found: https://raw.githubusercontent.com/msys2/setup-msys2/v3/action.yml
2024/12/09 16:30:58 Checking https://raw.githubusercontent.com/peaceiris/actions-gh-pages/v5/action.yml
2024/12/09 16:30:58 Not found: https://raw.githubusercontent.com/ncipollo/release-action/v2/action.yml
2024/12/09 16:30:58 Checking https://raw.githubusercontent.com/peter-evans/create-pull-request/v8/action.yml
2024/12/09 16:30:58 Not found: https://raw.githubusercontent.com/nwtgck/actions-netlify/v4/action.yml
2024/12/09 16:30:58 Checking https://raw.githubusercontent.com/preactjs/compressed-size-action/v3/action.yml
2024/12/09 16:30:59 Not found: https://raw.githubusercontent.com/octokit/request-action/v3.x/action.yml
2024/12/09 16:30:59 Checking https://raw.githubusercontent.com/pulumi/actions/v7/action.yml
2024/12/09 16:30:59 Not found: https://raw.githubusercontent.com/peaceiris/actions-gh-pages/v5/action.yml
2024/12/09 16:30:59 Checking https://raw.githubusercontent.com/pypa/gh-action-pypi-publish/release/v2/action.yml
2024/12/09 16:30:59 Not found: https://raw.githubusercontent.com/peter-evans/create-pull-request/v8/action.yml
2024/12/09 16:30:59 Checking https://raw.githubusercontent.com/reviewdog/action-actionlint/v2/action.yml
2024/12/09 16:30:59 Not found: https://raw.githubusercontent.com/preactjs/compressed-size-action/v3/action.yml
2024/12/09 16:30:59 Checking https://raw.githubusercontent.com/reviewdog/action-eslint/v2/action.yml
2024/12/09 16:30:59 Not found: https://raw.githubusercontent.com/pulumi/actions/v7/action.yml
2024/12/09 16:30:59 Checking https://raw.githubusercontent.com/reviewdog/action-golangci-lint/v3/action.yml
2024/12/09 16:30:59 Not found: https://raw.githubusercontent.com/pypa/gh-action-pypi-publish/release/v2/action.yml
2024/12/09 16:30:59 Checking https://raw.githubusercontent.com/reviewdog/action-hadolint/v2/action.yml
2024/12/09 16:30:59 Not found: https://raw.githubusercontent.com/reviewdog/action-actionlint/v2/action.yml
2024/12/09 16:30:59 Checking https://raw.githubusercontent.com/reviewdog/action-misspell/v2/action.yml
2024/12/09 16:30:59 Not found: https://raw.githubusercontent.com/reviewdog/action-eslint/v2/action.yml
2024/12/09 16:30:59 Checking https://raw.githubusercontent.com/reviewdog/action-rubocop/v3/action.yml
2024/12/09 16:30:59 Not found: https://raw.githubusercontent.com/reviewdog/action-hadolint/v2/action.yml
2024/12/09 16:30:59 Checking https://raw.githubusercontent.com/reviewdog/action-shellcheck/v2/action.yml
2024/12/09 16:30:59 Not found: https://raw.githubusercontent.com/reviewdog/action-golangci-lint/v3/action.yml
2024/12/09 16:30:59 Checking https://raw.githubusercontent.com/reviewdog/action-tflint/v2/action.yml
2024/12/09 16:30:59 Not found: https://raw.githubusercontent.com/reviewdog/action-rubocop/v3/action.yml
2024/12/09 16:30:59 Checking https://raw.githubusercontent.com/rhysd/action-setup-vim/v2/action.yml
2024/12/09 16:30:59 Not found: https://raw.githubusercontent.com/reviewdog/action-misspell/v2/action.yml
2024/12/09 16:30:59 Checking https://raw.githubusercontent.com/ridedott/merge-me-action/v3/action.yml
2024/12/09 16:30:59 Not found: https://raw.githubusercontent.com/reviewdog/action-tflint/v2/action.yml
2024/12/09 16:30:59 Checking https://raw.githubusercontent.com/rtCamp/action-slack-notify/v3/action.yml
2024/12/09 16:30:59 Not found: https://raw.githubusercontent.com/reviewdog/action-shellcheck/v2/action.yml
2024/12/09 16:30:59 Checking https://raw.githubusercontent.com/ruby/setup-ruby/v2/action.yml
2024/12/09 16:31:00 Not found: https://raw.githubusercontent.com/rhysd/action-setup-vim/v2/action.yml
2024/12/09 16:31:00 Checking https://raw.githubusercontent.com/shivammathur/setup-php/v3/action.yml
2024/12/09 16:31:00 Not found: https://raw.githubusercontent.com/ridedott/merge-me-action/v3/action.yml
2024/12/09 16:31:00 Checking https://raw.githubusercontent.com/softprops/action-gh-release/v3/action.yml
2024/12/09 16:31:00 Not found: https://raw.githubusercontent.com/rtCamp/action-slack-notify/v3/action.yml
2024/12/09 16:31:00 Checking https://raw.githubusercontent.com/subosito/flutter-action/v3/action.yaml
2024/12/09 16:31:00 Not found: https://raw.githubusercontent.com/ruby/setup-ruby/v2/action.yml
2024/12/09 16:31:00 Checking https://raw.githubusercontent.com/treosh/lighthouse-ci-action/v13/action.yml
2024/12/09 16:31:00 Not found: https://raw.githubusercontent.com/shivammathur/setup-php/v3/action.yml
2024/12/09 16:31:00 Checking https://raw.githubusercontent.com/wearerequired/lint-action/v3/action.yml
2024/12/09 16:31:00 Not found: https://raw.githubusercontent.com/softprops/action-gh-release/v3/action.yml
2024/12/09 16:31:00 Checking https://raw.githubusercontent.com/peaceiris/actions-hugo/v4/action.yml
2024/12/09 16:31:00 Not found: https://raw.githubusercontent.com/subosito/flutter-action/v3/action.yaml
2024/12/09 16:31:00 Checking https://raw.githubusercontent.com/actions/attest-build-provenance/v3/action.yml
2024/12/09 16:31:00 Not found: https://raw.githubusercontent.com/treosh/lighthouse-ci-action/v13/action.yml
2024/12/09 16:31:00 Checking https://raw.githubusercontent.com/actions/add-to-project/v2.0.0/action.yml
2024/12/09 16:31:00 Not found: https://raw.githubusercontent.com/wearerequired/lint-action/v3/action.yml
2024/12/09 16:31:00 Checking https://raw.githubusercontent.com/octokit/graphql-action/v3.x/action.yml
2024/12/09 16:31:00 Not found: https://raw.githubusercontent.com/peaceiris/actions-hugo/v4/action.yml
2024/12/09 16:31:00 Not found: https://raw.githubusercontent.com/actions/attest-build-provenance/v3/action.yml
2024/12/09 16:31:00 Not found: https://raw.githubusercontent.com/actions/add-to-project/v2.0.0/action.yml
2024/12/09 16:31:00 Not found: https://raw.githubusercontent.com/octokit/graphql-action/v3.x/action.yml
2024/12/09 16:31:00 Done detecting new versions in 90 repositories
No new release was found
```

</details>

**Review by rhysd — APPROVED** [review]

**Comment by rhysd** [comment]
Thanks!

**Comment by massongit** [comment]
>Due to not updated `popular_actions.go`, actionlint reports false positive on my repo.

According to https://github.com/reviewdog/action-actionlint/issues/149, the above issue is still continuing.
Therefore, please release this PR.

**Pull request merged**